src/quorum.c

```diff
@@ -227,7 +230,8 @@ static int read_quorum_blocks(struct super_block *sb, struct list_head *blocks)
 
 	for (i = 0; i < SCOUTFS_QUORUM_BLOCKS; i++) {
 		brelse(bh);
-		bh = sb_bread(sb, SCOUTFS_QUORUM_BLKNO + i);
+		bh = __bread(sbi->meta_bdev, SCOUTFS_QUORUM_BLKNO + i,
+			     sbi->meta_bdev->bd_block_size);
```

Comment:
and I guess we missed the use of _SM_SIZE here when updating the getblks(),

Reply:
doh!